lib/rubocop/cop/style/string_concatenation.rb

```diff
@@ -15,7 +15,13 @@ module Style
       # lines, this cop does not register an offense; instead,
       # `Style/LineEndConcatenation` will pick up the offense if enabled.
       #
-      # @example
+      # Two modes are supported:
```

Comment:
You should also mention how this ties to the cop safety - e.g. with the `Pathname` example.